pdns/dnsdistdist/meson-dist-script.sh

```diff
@@ -25,6 +25,8 @@ echo Running autoreconf -vi so distfile is still usable for autotools building
 # Run autoconf for people using autotools to build, this creates a configure sc
 autoreconf -vi
 rm -rf "$MESON_PROJECT_DIST_ROOT"/autom4te.cache
+echo Updating the version of the Rust library to ${BUILDER_VERSION}
```

Comment:
I think it would be a good things to error exit if `$BUILDER_VERSION` is not set, as `meson dist` can also be called not from the builder script.

Reply:
`set-configure-ac-version.sh` will die first in that case, but I guess I would be nice to have an actual check before that. Will do.